cni-plugin/integration/testutil/test_util.go

```diff
@@ -77,10 +175,14 @@ func (r *TestRunner) CheckCNIPluginIsLast() error {
 	}
 
 	plugins := conflist["plugins"].([]interface{})
-	lastPlugin := plugins[len(plugins)-1].(map[string]any)
-	if lastPlugin["name"] != "linkerd-cni" {
+	linkerdPlugin := plugins[len(plugins)-1].(map[string]any)
```

Comment:
Do you think we could use a comment here? I wanted to suggest it but tbh it's pretty self-explanatory if we look at the error message on L180.

Reply:
comment added!